src/imitation/algorithms/adversarial.py

```diff
@@ -1,18 +1,17 @@
 import os
 from functools import partial
-from typing import Callable, Optional, Sequence
+from typing import Callable, Mapping, Optional, Type
 from warnings import warn
 
 import numpy as np
 import tensorflow as tf
 import tqdm
-from stable_baselines.common.base_class import BaseRLModel
+from stable_baselines.common import base_class
 from stable_baselines.common.vec_env import VecEnv, VecNormalize
```

Comment:
Nit: since we seem to have changed everything else in this file to be `module.member`, change to import `vec_env` and then use `vec_env.VecEnv`, `vec_env.VecNormalize`?

Reply:
done